src/org/rascalmpl/repl/rascal/RascalReplServices.java

```diff
@@ -113,6 +122,27 @@ public ICommandOutput handleInput(String input) throws InterruptedException, Sto
         }
     }
 
+    private ICommandOutput createCancelledOutput() {
+        return new IErrorCommandOutput() {
```

Comment:
Hmm, I think I've seen this pattern a few times now. we should make a small helper class that wraps an `IErrorCommandOutput` around an `IOutputPrinter`